app/file.go

```diff
@@ -453,6 +461,25 @@ func getImageOrientation(input io.Reader) (int, error) {
 	}
 }
 
+func (a *App) genericThumbnailHandler() (*model.AppError) {
```

Comment:
This failed on style check. Should change to 
```
func (a *App) genericThumbnailHandler() *model.AppError {
```

Reply:
Thanks, I removed the parentheses.